packages/uiweb/src/lib/components/chat/ChatProfile/GroupInfoModal.tsx

```diff
@@ -120,18 +124,175 @@ const PendingMembers = ({
   }
 };
 
+const dummyConditonsData: ConditionArray[] = [
+  [{ operator: 'any' }],
+  [
+    {
+      type: 'PUSH',
+      category: 'ERC20',
+      subcategory: 'holder',
+      data: {
+        contract: 'eip155:1:0xf418588522d5dd018b425E472991E52EBBeEEEEE',
+        amount: 1,
+        decimals: 18,
+      },
+    },
+  ],
+  [
+    { operator: 'all' },
+    {
+      type: 'PUSH',
+      category: 'ERC20',
+      subcategory: 'holder',
+      data: {
+        contract: 'eip155:137:0x58001cC1A9E17A20935079aB40B1B8f4Fc19EFd1',
+        amount: 1,
+        decimals: 18,
+      },
+    },
+    {
+      type: 'PUSH',
+      category: 'ERC721',
+      subcategory: 'holder',
+      data: {
+        contract: 'eip155:137:0x58001cC1A9E17A20935079aB40B1B8f4Fc19EFd1',
+        amount: 1,
+        decimals: 18,
+      },
+    },
+    {
+      type: 'GUILD',
+      category: 'ROLES',
+      subcategory: 'DEFAULT',
+      data: {
+        id: '1',
+        role: '346243',
+        comparison: 'all',
+      },
+    },
+  ],
+  [
+    { operator: 'any' },
+    {
+      type: 'PUSH',
+      category: 'INVITE',
+      subcategory: 'DEFAULT',
+      data: {
+        inviterRoles: 'ADMIN',
+      },
+    },
+    {
+      type: 'PUSH',
+      category: 'INVITE',
+      subcategory: 'DEFAULT',
+      data: {
+        inviterRoles: 'OWNER',
+      },
+    },
+  ],
+];
+
+const dummySingleCondtionData: ConditionArray[] = dummyConditonsData[2].map((criteria) => [
+  criteria,
+]);
+
+interface ConditionsInformationProps{
+  theme: IChatTheme;
+  groupInfo?: IGroup | null;
+}
+
+export const ConditionsInformation = ({theme, groupInfo}: ConditionsInformationProps) => {
+  return (
+    <Section margin='15px 0px 0px 0px' flexDirection='column'>
+      <Section margin='0px 0px 15px 0px'>
+      {
+        (groupInfo?.rules?.chat?.conditions || groupInfo?.rules?.entry?.conditions) && (
+          <PublicEncrypted theme={theme}>
```

Comment:
the GateGroup section can be reused same as groupInformation component

Reply:
<img width="429" alt="image" src="https://github.com/ethereum-push-notification-service/push-sdk/assets/42746736/26fc7681-de63-4ea2-a925-beaef17bb435">
this is not there